lib/iStats/cpu.rb

```diff
@@ -27,10 +26,15 @@ def all
 
       # Print CPU temperature with sparkline
       #
-      def cpu_temperature
-        t = get_cpu_temp
-        thresholds = [50, 68, 80, 90]
-        puts "CPU temp: #{t}#{Symbols.degree}C  " + Printer.gen_sparkline(t, thresholds)
+      def cpu_temperature        
```

Comment:
Don't think there's a need for a default config. The default sensor for CPU is hardcoded in code (current version). If someone runs the command and there's no information for CPU temp, we can display a message and suggests the user runs a scan and sets the sensor key. It's a bit more completed but for most people the default works so I don't want to make it hard for them. So the flow would become: 
1. Print CPU stats
2. No information found
3. Display help message that suggests running a scan and setting the sensor key in config.
4. User runs scan
5. User sets sensor key using a newly added command.